src/boot.zig

```diff
@@ -21,7 +21,7 @@ pub const ft_options: @import("ft/ft.zig").Options = .{
     .logFn = @import("logger.zig").kernel_log,
 };
 
-export fn _entry() linksection(".bootstrap_code") callconv(.Naked) noreturn {
+export fn _start() linksection(".bootstrap_code") callconv(.Naked) noreturn {
```

Comment:
why change the name?

Reply:
For any reason, ld seems broken on zig 0.12:

![image](https://github.com/riblanc/kfs/assets/59930230/92c3fd7e-02ff-44d4-954b-5a8675213197)

Reply:
https://github.com/ziglang/zig/issues/18482#issuecomment-1881752824

Reply:
I finally went back to ``_entry``, now defining the name of the entry symbol in the build script.